packages/block-library/src/comments-pagination-previous/edit.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, PlainText } from '@wordpress/block-editor';
+
+const arrowMap = {
+	none: '',
+	arrow: '←',
+	chevron: '«',
+};
+
+export default function CommentsPaginationPreviousEdit( {
+	attributes: { label },
+	setAttributes,
+	context: { paginationArrow },
+} ) {
+	const displayArrow = arrowMap[ paginationArrow ];
+	return (
+		<a
+			href="#comments-pagination-previous-pseudo-link"
+			onClick={ ( event ) => event.preventDefault() }
+			{ ...useBlockProps() }
+		>
+			{ displayArrow && (
+				<span
+					className={ `wp-block-query-pagination-previous-arrow is-arrow-${ paginationArrow }` }
```

Comment:
I wanted to reiterate my concern shared while reviewing the Next Page block that sharing the same class name with an unrelated block is risky. The whole point of using the [BEM naming](http://getbem.com/naming/) is to ensure that style changes are isolated to components or blocks. In the case of the Query Loop block and the Comments Query Loop block, we might even end up in the situation where one is nested into another making styling harder than usual. A blog post list would be an immediate example, see here https://make.wordpress.org/core/:

<img width="1179" alt="Screenshot 2021-12-21 at 10 17 28" src="https://user-images.githubusercontent.com/699132/146904353-72e91b6c-f86d-470c-8c1e-c7c73b010a90.png">

As of today, we have the same design for Query Pagination blocks and Comments Pagination blocks, but it doesn't have to be always true. You might want to use different styles at some point.

Reply:
We can solve it ASAP by creating a `get_comment_pagination_arrow()` similar to `get_query_pagination_arrow()`, or adding a new parameter to `get_query_pagination_arrow()` like $is_comment_pagination. 
What do you think?

Reply:
See my comment on the Next Page block: https://github.com/WordPress/gutenberg/pull/36562#discussion_r771292001. I'd be in favor of a completely new helper. It's something that we should tackle separately - not in this PR - because it impacts also the other block.